recipes/moola/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "moola" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b5936cef0d2a29432784dea9e4813e3653d5277a66bac67bf5d89ba912ec7ce
+  patches:
+    - imports.patch
```

Comment:
Was this submitted upstream? Guessing PR ( https://github.com/funsim/moola/pull/8 ). Could we add a comment above this line citing that?

Reply:
added note that all patches are merged upstream.